src/app/[locale]/(logged-out)/privacy-policy/components/PrivacyPolicy/privacy_policy.md

```diff
@@ -0,0 +1,121 @@
+# SOURSD Privacy Policy
+
+## Purpose
+
+This privacy policy provides information on how Health Data Research UK (“HDR UK”, “we”, “us” or “our”) collects and processes your personal data. It also describes your data protection rights, including a right to object to some of the processing which HDR UK carries out. More information about your rights, and how to exercise them, is set out in the “What rights do I have?” section.
+
+This privacy policy provides information on how Health Data Research UK (“HDR UK” “we”, “us” or “our”) collects and processes your personal data through the Safe User and Organisation Registry for Sensitive Data (SOURSD) web-based tool. It also describes your data protection rights, including a right to opt out to some of the processing which HDR UK carries out.
```

Comment:
This paragraphs is a duplicate of the one above, or at least seems very close.

Reply:
pfft didnt look at it - it was what they signed off.. https://hdruk.app.box.com/file/1817049885348?s=vfo7anivijhhm3znd33sa80z232gcea5
🤷